### PR TITLE
Responsiveness Fixed #1605

### LINKED ISSUE #1605: [Fix] Make About page Responsive
### Details

Like now when we try to view it on the mobile the background kind of left some spaces at the border , so I can fix that please assign this issue to me under gssoc'23 thank you. 

### Screenshots

Screenshot (181)

### Type of Contribution

- [ ] Update to an existing Animated Button
- [ ] Adding a new Button
- [X] Resolving a bug
- [ ] Proposal to the Repository
- [ ] Changes related to documentation or README.md
- [ ] Other Changes

### Checklist

- [X] I have checked the existing [issues](https://github.com/Spyware007/Animating-Buttons/issues)
- [X] I have read the [Contributing Guidelines](https://github.com/Spyware007/Animating-Buttons/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Hemant2335**
## Fixes Issue

Closes #1605 

## Changes proposed

Changed Two files and have made their width changed from vh values to max-content  for make sure to fill the screen completely

## Screenshots
Screenshot (185)

## Checklist

- [ x] You've linked this PR to the correct issue.
- [ x] You have checked that the code is working correctly.
- [ x] You ⭐️ the repository!

### CODE REVIEW

**Comment by adity2911** [comment]
Can you add Hamburger menu for smaller screens. 
there is a problem with this. After a certain screen size, the navbar disappears.

**Comment by Hemant2335** [comment]
Yes , I can do this and i think you wanted me to add a new features right ??

**Comment by adity2911** [comment]
yes

**Comment by Hemant2335** [comment]
ok I am on it will let you know once done !!

**Comment by Hemant2335** [comment]
Also can I use Tailwind ??

**Comment by adity2911** [comment]
sure

**Comment by Souravvmishra** [comment]
@Hemant2335 Any Updates?

**Comment by Hemant2335** [comment]
Yaa one more day please

**Comment by adity2911** [comment]
Resolve conflicts.
Also kindly share images of your working changes. Screen Recording will be better.

**Comment by Hemant2335** [comment]
Yaa Here's the video the UI might not be that great 

https://github.com/Spyware007/Animating-Buttons/assets/85656999/ebf373c4-22fd-4388-8f30-6c834a219c45

**Comment by Spyware007** [comment]
@Hemant2335 The navbar doesn't seem to be right, it should be a full-screen navbar like most of the websites have!

**Comment by Hemant2335** [comment]
you mean when clicked it should like cover the complete webpage

**Comment by adity2911** [comment]
Yes. A full page navbar is required.

**Comment by Hemant2335** [comment]
Yaa I have done the job please check

https://github.com/Spyware007/Animating-Buttons/assets/85656999/7f33ac61-f02b-4d7c-97e4-f83a777ee8f4

**Comment by Hemant2335** [comment]
Also I think there is some issue with that About Page

File: src/firebase/auth.js [hunk 1]
```diff
@@ -5,12 +5,12 @@ import { getFirestore } from "firebase/firestore";
 import { getAnalytics } from "firebase/analytics";
 
 const firebaseConfig = {
-  apiKey: process.env.REACT_APP_apiKey,
```
Comment (adity2911) [inline src/firebase/auth.js:8]: revert this change.

Comment (Hemant2335) [reply src/firebase/auth.js]: Yaa Resolved

**Comment by adity2911** [comment]
@Spyware007 Kindly check

**Comment by Souravvmishra** [comment]
@Hemant2335 It Broke In Phone, Please Check

**Comment by Hemant2335** [comment]
It works fine in the responsive view and  , I think you are viewing in zoomed mode try zooming out please
Screenshot (188)

**Comment by Souravvmishra** [comment]
https://github.com/Spyware007/Animating-Buttons/assets/89072785/15f377f6-5b72-4b79-a5b8-211e3a182350

**Comment by Souravvmishra** [comment]
@Hemant2335 @Spyware007

**Comment by Hemant2335** [comment]
Please Check it now 
Screenshot (190)

**Comment by Souravvmishra** [comment]
Overflow Issue

**Comment by Spyware007** [comment]
Its looking odd
Don't merge it!

Can you fix it @Hemant2335

**Comment by Hemant2335** [comment]
I can make the UI good don't worry about it !!

but I have changed the responsiveness back to what it used to be and  I want to restrain this pr to creating a mobile menu.

**Comment by Souravvmishra** [comment]
@Spyware007 Please Look Into This. Also For The Responsive Navbar, Please Cheeck #1684
